plugins/inputs/tail/tail.go

```diff
@@ -162,6 +163,54 @@ func (t *Tail) Start(acc telegraf.Accumulator) error {
 	return err
 }
 
+func (t *Tail) getSeekInfo(file string, fromBeginning bool) (*tail.SeekInfo, error) {
+	var seek *tail.SeekInfo
+
+	seekStart := &tail.SeekInfo{
+		Whence: 0,
+		Offset: 0,
+	}
+	seekEnd := &tail.SeekInfo{
+		Whence: 2,
+		Offset: 0,
+	}
+
+	if fromBeginning && t.ReadStart == "" {
+		return seekStart, nil
+	}
+
```

Comment:
Please handle this in `Init()`  and afterwards only rely on `ReadStart`. This eases removing of the deprecated `from_beginning` setting later.